CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 # Enable all warnings.
 add_compile_options(-Wall -Werror)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-  add_compile_options(-Wno-multichar -Wno-sign-compare)
+  add_compile_options(-Wno-multichar -Wno-sign-compare -fno-strict-aliasing)
```

Comment:
It caused build failures.  There are some parts that would try to cast the address of doubles as the address of uint64_t's, which caused this to trip.  __ABPWriteDate() is one such place this happened.